docs/pages/guides/sharing-before-shipping.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Sharing your app before you ship
+---
+
+Ultimately, you will want to publish your application to the App Store and Play Store to access the opportunities for wider distribution and to provide a familiar installation process for your users.  While you are developing though, you'll want to let other members of your team and your potential users see the work in progress to gather feedback and make sure you're building a quality product.
+
+## Expo Go
+
+If you are using the Expo Managed workflow, you can share your application with your team through the Expo Go app.
+
+1. Publish the latest version of your project to Expo's servers by running `expo publish`.
+2. [Invite your teammate](https://expo.io/[account]/[project]/settings/members) to the Expo account that owns the project.
+3. Have them [download the Expo Go app](https://expo.io/client) onto their device.
+4. They'll be able to open your application from the Profile tab of their Expo Go app.
```

Comment:
Should we also mention that you can share the manifest link `exp.host/@username/project/` with them and it'll open the project (can use QR code as well)